src/Compilers/CSharp/Portable/BoundTree/BoundInterpolatedString.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class BoundInterpolatedString
+    {
+        public const string AppendFormattedMethod = "AppendFormatted";
+        public const string AppendLiteralMethod = "AppendLiteral";
```

Comment:
Could be useful as public API in `WellKnownMemberNames`?

Reply:
If we get a request for it, we could potentially add it.